modules/apps/portal-vulcan/portal-vulcan-test/src/testIntegration/java/com/liferay/portal/vulcan/internal/graphql/servlet/test/GraphQLServletTest.java

```diff
@@ -62,9 +67,19 @@ public void test() throws Exception {
 			testDTOSimpleClassName, new GraphQLField("field"),
 			new GraphQLField("_id"));
 
+		GraphQLField queryGraphQLField = new GraphQLField(
+			"query", graphQLField);
+
+		String body = JSONUtil.put(
```

Comment:
I think this is a bit confusing. Why don't we just modify the invoke method from the parent class to use the HTTPTestUtil?